lib/stats/stat.js

```diff
@@ -12,7 +15,14 @@ export default class Stat {
     assert(config.workerNodeType, 'Worker instance type is required');
     assert(config.provisionerId, 'Provisioner ID is required');
 
-    this.influx = new base.stats.Influx(config.influx);
+    if (config.influx.connectionString) {
```

Comment:
nice to have this spit out to the console when debugging rather than seeing the connection errors when testing locally.  Could change this to emit somethign on process instead and then we could listen for those and test the stats as well...